src/main/java/com/configcat/PercentageOption.java

```diff
@@ -0,0 +1,38 @@
+package com.configcat;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents a percentage option.
+ */
+public class PercentageOption {
+
+    /**
+     * A number between 0 and 100 that represents a randomly allocated fraction of the users.
```

Comment:
I'd rather move these docs comments from private fields to the public accessors.